v2/rscg_examples_site/docs/RSCG-Examples/validly.md

```
@@ -0,0 +1,382 @@
+![Alt text](https://github.com/Hookyns.png)
+:::
+
+### Original Readme
+:::note
+
+# Validly
+
+[![NuGet](https://img.shields.io/nuget/v/Validly)](https://www.nuget.org/packages/Validly)
+[![License MIT](https://img.shields.io/badge/License-MIT-brightgreen?style=flat-square)](https://opensource.org/licenses/MIT)
+[![Support Ukraine](https://img.shields.io/badge/Support-Ukraine-FFDD00?labelColor=0057B7)](https://donate.redcross.org.uk/appeal/ukraine-crisis-appeal)<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![All Contributors](https://img.shields.io/badge/all_contributors-5-orange.svg?style=flat-square)](#contributors-)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)